packages/cmf/src/sagas/component.js

```diff
@@ -0,0 +1,20 @@
+import invariant from 'invariant';
+import { fork, cancel, take, takeEvery } from 'redux-saga/effects';
+import CONST from '../constant';
+import { get } from '../saga';
+
+export function* onSagaStart(action) {
+	const saga = get(action.saga);
+	if (!saga) {
+		invariant(process.env.NODE_ENV !== 'production', `The saga ${action.saga} is not registred`);
+	} else {
+		const task = yield fork(saga);
```

Comment:
It might be a dumb question, but do we really always want to do a fork of the saga ?